x/incentives/keeper/gauge.go

```diff
@@ -66,25 +79,23 @@ func (k Keeper) SetGaugeWithRefKey(ctx sdk.Context, gauge *types.Gauge) error {
 
 	curTime := ctx.BlockTime()
 	timeKey := getTimeKey(gauge.StartTime)
+	ActiveOrUpcomingGauge := gauge.IsActiveGauge(curTime) || gauge.IsUpcomingGauge(curTime)
+
 	if gauge.IsUpcomingGauge(curTime) {
-		if err := k.addGaugeRefByKey(ctx, combineKeys(types.KeyPrefixUpcomingGauges, timeKey), gauge.Id); err != nil {
-			return err
-		}
-		if err := k.addGaugeIDForDenom(ctx, gauge.Id, gauge.DistributeTo.Denom); err != nil {
-			return err
-		}
+		CombinedKeys := combineKeys(types.KeyPrefixUpcomingGauges, timeKey)
```

Comment:
Are these variables intentionally capitalized?

Reply:
Oops, I think I should have camel-cased them instead judging by the surrounding context.  I will fix this and then push the necessary changes!